core-bundle/src/Command/SymlinksCommand.php

```diff
@@ -116,6 +116,8 @@ protected function executeLocked(InputInterface $input, OutputInterface $output)
     private function generateSymlinks(): void
     {
         $fs = new Filesystem();
+        $uploadPath = $this->getContainer()->getParameter('contao.upload_path');
+        $bundles = $this->getContainer()->getParameter('kernel.bundles');
```

Comment:
Upload path is injected in the constructor, why do you get it here again?
And we could also inject the bundles parameter then :)